packages/graphql-fixtures/package.json

```diff
@@ -24,9 +24,9 @@
     "node": ">=12.14.0"
   },
   "dependencies": {
+    "@faker-js/faker": "^5.5.3",
     "@shopify/useful-types": "^3.0.5",
-    "@types/faker": "^4.1.2",
-    "faker": "^4.1.0",
+    "@types/faker": "^5.5.9",
```

Comment:
This feels like it shall be kinda janky - we're depending on `@faker-js/faker` but we're not shipping any types for that package. We currently rely on your faker.d.ts file to say "hey they types for `@faker-js/faker` are the same as `faker`" but that would only apply to the quilt repo, not in any other project.

Reply:
I hear you about this feedback and somewhat agree with the concern. However, do note that I'm not changing any of the existing behaviour regarding how `graphql-fixtures` is exporting the `@types/faker` package. I'm simply just preserving existing behaviour, except with a necessary upgrade.

May I suggest that we hold off on any further changes to this until Quilt upgrades to Faker v6? This way, Quilt can take advantage of having Faker being rewritten into TypeScript and thus not requiring a DefinitelyTyped dependency anymore.

Reply:
My fear was that we have publicly exposed function - `createFiller` whose type signature depends types that come from faker - and now that we're using `@faker-js/faker` instead of `faker` consumers won't know where to find the types for `@faker-js/faker`. We provide types for `@faker-js/faker` in this repo in the modules.d.ts file, but there's nothing teaching the published package that.

[Looking into the the types](https://unpkg.com/browse/graphql-fixtures@1.1.5/build/ts/fill.d.ts) it doesn't seem like our our exports's parameters or return types don't actually depend upon faker, so I think we get away with this.